tools/docker.sumologic-powershell-sdk/profile.ps1

```diff
@@ -0,0 +1,36 @@
+Import-Module ./SumoLogic.psm1
+
+if ($env:SHOW_CONNECTION_INFO -eq $true) {
+    write-host "Available commands in sumologic loaded module are:"
+    get-command -Module Sumologic
+}
+
+if ($Env:SUMO_SESSION) {
+    if (!$Env:SUMO_DEPLOYMENT -or !$Env:SUMO_ACCESS_ID -or !$Env:SUMO_ACCESS_KEY  ) { 
```

Comment:
Looks like no matter `true` or `false` in `$env:SUMO_SESSION`, all 3 vars are required? Maybe we should move this prompt to outside?